FirebaseAppDistribution/Sources/Public/FIRAppDistribution.h

```diff
@@ -12,13 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#import "FIRAppDistributionRelease.h"
```

Comment:
This could be changed to a forward declaration since it's only the type being used (`@class FIRAppDistributionRelease;`) instead of importing.

Reply:
done